docs/md/core.md

```diff
@@ -873,7 +873,7 @@ where constant expressions are required.
 
 As long as the list remains unchanged, identifiers are also guaranteed to be
 stable across different runs. In case they have been used in a production
-environment and a type has to be removed, one can just use a placeholder to
+environment, and a type has to be removed. One can just use a placeholder to
```

Comment:
Oh, I see the misunderstanding now 😅 this is more on the line: _One can do xxx in case they have been uses ..., and a type as to be removed_. I flipped everything upside down in perfect Italian style, sorry for that. 🙏